File: DayTwo/java script/level one/tasknine.js
const readline = require('readline');

rl = readline.createInterface({
  input: process.stdin,
  output: process.stdou});

rl.question('Enter the price of the item: ', (price) => {
  const tax = parseFloat(price) * 0.10;
  const total = parseFloat(price) + tax;

  console.log(`Tax: ${tax}`);
  console.log(`Total: ${total}`);
});

File: DayTwo/java/level one/TaskNine.java
import java.util.Scanner;

public class TaxCalculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the price of the item: ");
        double price = scanner.nextDouble();

        double tax = price * 0.10;
        double total = price + tax;

        System.out.println("Tax: " + tax);
        System.out.println("Total: " + total);
    }
}

